app/server/lib/DocApi.ts

```diff
@@ -412,6 +413,22 @@ export class DocWorkerApi {
       }
 
       if (tableId !== undefined) {
+        if (columnIds !== undefined && columnIds !== null && columnIds !== '') {
+          if (tableId !== currentTableId && currentTableId !== undefined && currentTableId !== null) {
```

Comment:
I guess it's safe to assume that `currentTableId` have to also be a non-empty string?

```suggestion
          if (tableId !== currentTableId && currentTableId) {
```